Makefile

```diff
@@ -15,8 +22,8 @@ provision-cargateway: ./fixtures.car
 	car -c ./fixtures.car &
 
 provision-kubo:
-	find ./fixtures -name '*.car' -exec ipfs dag import {} \;
-	find ./fixtures -name '*.ipns-record' -exec sh -c 'ipfs routing put --allow-offline /ipns/$$(basename -s .ipns-record "{}") "{}"' \;
+	find ./fixtures -name '*.car' -exec ipfs dag import --stats --pin-roots=false {} \;
```

Comment:
:information_source:  pinning blocks on things like missing blocks, we don't need it as GC is not enabled by default in Kubo